### ISSUE TITLE
Model Evaluation running on Azure

### ISSUE THREAD

**Issue by mikelle-rogers** [orcasound/aifororcas-livesystem#109]

**Comment by mikelle-rogers** [comment]
@tsuize do you want to sync so that we make sure we aren't duplicating efforts?

**Comment by tsuize** [comment]
@mikelle-rogers Yes, definitely. I'm not familiar with Azure. Right now I'm trying to run the data preprocess scripts on it and exploring Azure. Basically, I have not much going on with the actual Model evaluation...

**Comment by mikelle-rogers** [comment]
Okay, I kind of started working on something else. How has it been going for you?
